src/Components/Web.JS/Microsoft.AspNetCore.Components.Web.JS.npmproj

```diff
@@ -30,27 +30,6 @@
     <BuildOutputFiles Condition=" '$(Configuration)' != 'Debug' " Include="dist\Release\blazor.webview.js" />
```

Comment:
Is there something blocking us from deleting the .npmproj's? I'm fine leaving it as a follow-up, just want to know the state

Reply:
No, I just didn't want to make this even bigger, I plan to have a separate PR just for cleaning up stuff